main.go

```diff
@@ -54,16 +64,18 @@ func main() {
 		})
 
 		//	await
-		syncResult1 := GlobalEventLoop.Await(GetUserName(4))
-		fmt.Println("4 : user:", syncResult1)
+		syncResult1, err := GlobalEventLoop.Await(GetUserName(4))
+		fmt.Println("4 : user:", syncResult1, err)
 
-		syncResult2 := GlobalEventLoop.Await(GetUserName(1))
-		fmt.Println("1 : user:", syncResult2)
+		syncResult2, err := GlobalEventLoop.Await(GetUserName(1))
+		fmt.Println("1 : user:", syncResult2, err)
```

Comment:
Add this for more clarity 
```go
fmt.Sprintf("value: %v, err: %v", syncResult1, err)
```

Reply:
Pass 0 as id in GetUserName so we can test out the error handling functionally of Await
```go
GetUserName(0)
```